### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by kasrahabib** [seart-group/ghs#21] [Q&A]
Hi @emadpres,

Is there a way to crawl the Feature Requests for all projects???
Thanks in advance!

**Comment by emadpres** [comment]
Hi @kasrahabib 
Can you please clarify for me what you mean by "Feature Request"? Can you provide me an example?

**Comment by kasrahabib** [comment]
By the "Feature Requests," I mean a functionality recommendation through a user for a software project; in the case of GitHub, maybe on a repository of a project. 

Take for example, I use a repository, and I feel a functionality to be added then I will write a request to the owner if he/she is interested in adding it to their project.

**Comment by emadpres** [comment]
@kasrahabib With our platform (https://seart-ghs.si.usi.ch/) you can search for repositories with certain criteria, like GitHub repositories having at least 20 contributors. So the final output is always a list of repositories (with detailed information about them).

Close to what you are looking for, you can search for all repositories that have at least one _issue_ labeled `feature-request`. From there, you can write a follow-up script and extract such issues from each repository using GitHub API.

**Comment by kasrahabib** [comment]
@emadpres,

Cheers to that!
Had a quick look at it, that works!!!!

Thanks!

**Comment by kasrahabib** [comment]
@emadpres one more thing I forgot to add before closing the issue.
I can't harvest the issue text itself with this tool can I?

**Comment by emadpres** [comment]
@kasrahabib happy to see you got your answer.
Concerning your last question, as I said, the output is the list of repositories matching the criteria you specified. For each repo, you will find some detailed high-level info such as `number of commits`, `last commit date`, etc. Among such info, you will also have access to the **full list of labels used in a repo's issues** — but nothing more than the issue labels.
